openedx/core/djangoapps/programs/utils.py

```diff
@@ -151,7 +184,12 @@ def progress(self, programs=None, count_only=True):
                 if self._is_course_complete(course):
                     completed.append(course)
                 elif self._is_course_in_progress(course):
-                    in_progress.append(course)
+                    course_in_progress = self.does_course_still_qualify_as_in_progress(now, enrolled_run_modes, course)
```

Comment:
Does the microsecond difference in function calls warrant passing `now` as an argument?